app/schemas/sign_up.js

```diff
@@ -0,0 +1,24 @@
+const { body } = require('express-validator');
+
+module.exports = () => [
+  body('name')
+    .not()
+    .trim()
+    .isEmpty()
```

Comment:
Instead of using `not()` and `isEmpty()` you can use `notEmpty()` I think...
Also, maybe you can add the the type of the property, like 
```
body('name')
  .isString()
  .notEmpty()
  .exists() // to check that it is present on the body, with `notEmpty` is not enough I think
```